File: 1672/Solution_1672.cpp
#include <iostream>
#include <climits>
#include <vector>
#include <algorithm>
#include <numeric>

using namespace std;
class Solution {
    public:
        int maxWealthCount(vector<vector<int>>& accounts){
            int maxWealth = INT_MIN;
            for (const auto& account : accounts){
                int current_Wealth = accumulate(account.begin(),account.end(),0);
                maxWealth = max(maxWealth ,current_Wealth);
            }
            return maxWealth;
        }
};


int main (){
    vector<vector<int>> accounts = {
        {1,3,4},
        {3,5,6},
        {3,6,7}
    };

    Solution solu;

    int result = solu.maxWealthCount(accounts);
    cout << "result = " << result << endl;

    return 0;
}

File: 1672/Solution_1672.java
import java.util.Arrays;

public class Solution_1672 {
    public int maxWealthCount(int[][] accounts){
        int maxWealth = Integer.MIN_VALUE;
        for (int[] account : accounts){
            maxWealth = Math.max(maxWealth , Arrays.stream(account).sum()); 
        }
        return maxWealth;
    }

    public static void main(String[] args){
        int[][] acc= {{1,3,4},{2,4,5},{2,5,3}};

        Solution_1672 solu = new Solution_1672();

        int result = solu.maxWealthCount(acc);

        System.out.println("result = " + result );
    }
}
        
